content/en/blogs/deep-dive-into-kubernetes-logging.md

```diff
@@ -0,0 +1,141 @@
+---
+title: 'Deep Dive into Kubernetes Logging'  
+tag: 'Kubernetes, Logging, Fluent Operator'  
+keywords: Kubernetes, Logging, Fluent Operator  
+description: This article looks into the different types of Kubernetes logs needed for better observability as well as approaches to implement logging in Kubernetes.   
+createTime: '2022-03-23'  
+author: 'Yitaek Hwang'  
+snapshot: 'https://i.imgur.com/P77MbR6.png'
```

Comment:
@Felixnoo Why do we choose to save the images to `i.imgur.com`?

Reply:
Great catch. I added the images in [this commit](https://github.com/kubesphere/website/pull/2351/commits/41fc40a4dec88b1110167f67510ae3aecd8a7a31).